### PR TITLE
Bug/VMRay/Fixed: viewing all threat-indicators for a sampleID

### PR DESCRIPTION
**Pull request by obenacot**
… of viewing the first one.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/36974

## Description
export the whole indicator list to the War-Room Entry instead of only the first one.

### CODE REVIEW

**Review by yaakovi — CHANGES_REQUESTED** [review]
* update the PR's description
* update release notes

**Comment by obenacot** [comment]
Closed due to PR Duplicaty

File: Packs/VMRay/Integrations/VMRay/VMRay.py [hunk 1]
```diff
@@ -630,11 +630,11 @@ def get_threat_indicators_command():
...
 
         human_readable = tableToMarkdown(
-            'Threat indicators for sample ID: {}. Showing first indicator:'.format(
+            'Threat indicators for sample ID: {}:'.format(
                 sample_id
             ),
-            entry_context_list[0],
-            headers=['AnalysisID', 'Category', 'Classification', 'Operation'],
+            entry_context_list,
+            headers=['AnalysisID', 'Category', 'Classification', 'ID', 'Operation'],
```
Comment (yaakovi) [inline Packs/VMRay/Integrations/VMRay/VMRay.py:637]:
```suggestion
            headers=['ID', 'AnalysisID', 'Category', 'Classification', 'Operation'],
```

File: Packs/VMRay/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### VMRay
+- Enable mocking
```
Comment (yaakovi) [inline Packs/VMRay/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Fixed an issue where only the first indicator was shown in the War-Room entry for the ***vmray-get-threat-indicators*** command. 
```

**Review by yaakovi — COMMENTED** [review]
fill in the PR description

File: Packs/VMRay/ReleaseNotes/1_0_2.md (hunk 2 above)
Comment (amshamah419) [reply Packs/VMRay/ReleaseNotes/1_0_2.md:4]: 👍

**Review by yaakovi — APPROVED** [review]

**Pull request merged**